guava/src/module-info.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2008 The Guava Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Google Guava
+ */
+module com.google.common {
+  requires static jdk.unsupported;
+  requires transitive java.logging;
```

Comment:
`jdk.unsupported` may not be needed, since usages of `Unsafe` are only compiled in for pre-JDK9.
`requires transitive java.logging` needs a second look

Reply:
It's needed